docs/development/jump-menu.md

```diff
@@ -17,33 +17,50 @@ Add-ons can easily add custom items for the Jump Menu.
 
 NOTE: Sample File: https://gist.github.com/obfuscode/77d0135970c9ce8bc84139c127507bb1
```

Comment:
We should update this link to point to a gist we have access to.

Reply:
Removed this... especially since we have the generator now.